app/models/user.rb

```diff
@@ -10,7 +51,9 @@ class User < ApplicationRecord
   has_many :org_edits, through: :permissions, source: :organization
   has_many :prog_edits, through: :permissions, source: :program
 
+  accepts_nested_attributes_for :media, allow_destroy: true
+
   # Validations
-  validates :first_name, :last_name, :linkedin_id, :contact_url, :role, presence: true
```

Comment:
Can probably still have the presence validation on role since it is guaranteed to be set to at least a default now.